From: David Lutz <kpanic@hirnduenger.de>
Date: Mon, 12 Mar 2018 17:28:33 +0100
Subject: ramips: Fix WiFi-LEDs for TP-Link TL-WR841N v13 and TL-WR840N v4

diff --git a/target/linux/ramips/base-files/etc/board.d/01_leds b/target/linux/ramips/base-files/etc/board.d/01_leds
index 0a870d9f6c04e0a7c18c24beb207843915f2b250..f6fff5cff5896b32ef42478feee775c8c91add15 100755
--- a/target/linux/ramips/base-files/etc/board.d/01_leds
+++ b/target/linux/ramips/base-files/etc/board.d/01_leds
@@ -378,12 +378,12 @@ tiny-ac)
 	set_usb_led "$boardname:green:usb"
 	;;
 tl-wr840n-v4)
-	set_wifi_led "$boardname:green:wlan"
+        ucidef_set_led_wlan "wlan" "WLAN" "$boardname:green:wlan" "phy0tpt"
 	ucidef_set_led_switch "lan" "lan" "$boardname:green:lan" "switch0" "0x1e"
 	ucidef_set_led_switch "wan" "wan" "$boardname:green:wan" "switch0" "0x01"
 	;;
 tl-wr841n-v13)
-	set_wifi_led "$boardname:green:wlan"
+        ucidef_set_led_wlan "wlan" "WLAN" "$boardname:green:wlan" "phy0tpt"
 	ucidef_set_led_switch "lan1" "lan1" "$boardname:green:lan1" "switch0" "0x2"
 	ucidef_set_led_switch "lan2" "lan2" "$boardname:green:lan2" "switch0" "0x4"
 	ucidef_set_led_switch "lan3" "lan3" "$boardname:green:lan3" "switch0" "0x8"
